apps/portal/theme/templates/designer.jag

```diff
@@ -195,6 +198,7 @@
 <script src="<%=urlPrefix%>js/jquery-1.10.2.js"></script>
 <script src="<%=urlPrefix%>js/bootstrap.min.js"></script>
 <script src="<%=urlPrefix%>js/jquery-ui.js"></script>
+<script src="<%=urlPrefix%>js/cropper.js"></script>
```

Comment:
Move the scripts that can be moved to bottom of the page to speed up the page loading.